src/contractTest/java/uk/gov/hmcts/reform/professionalapi/provider/ProfessionalExternalUserControllerProviderTestConfiguration.java

```diff
@@ -38,6 +39,12 @@ public class ProfessionalExternalUserControllerProviderTestConfiguration extends
     @MockBean
     ServiceAuthFilter serviceAuthFilter;
 
+    @MockBean
+    UserConfiguredAccessRepository userConfiguredAccessRepository;
```

Comment:
userConfiguredAccessRepository, where is this used in test cases?

Reply:
added to a new test